lms/static/styles/components/_SubmitGradeForm.scss

```diff
@@ -0,0 +1,88 @@
+@use "../variables" as var;
+
+
+@keyframes gradeSaved {
+  from {
+    background-color: #58bf4b;
+  }
+  to {
+    background-color: transparent;
+  }
+}
+
+
+.SubmitGradeForm {
+  display: flex;
+  white-space: nowrap;
+  position: relative;
+
+  > button {
```

Comment:
@lyzadanger 
Wondering how you feel about this style here. I generally gravitate to more terse css, less classes if possible and I like expressing the dom tags in the sass when possible to help indicate what its for. However, this does contrast our sass conventions; however, how do you feel about direct descendent selectors to add specificity without having to create a class for each tag? It allows me to do a bit of nesting like this with low risk that the style would carry over to something it was not intended to?

Reply:
Thanks for highlighting this!

I think we should stick to our BEM conventions here; the challenge with direct-descendent selectors like this is that it creates a fragility: say you were to re-order some of the markup to move the button elsewhere in the component and it was suddenly no longer the direct child of `SubmitGradeForm`? Then your styles break. The nice thing about BEM, even though it can feel verbose at times, is that it sort of "encapsulates" an element's meaning into a class that can move about with that element, irrespective of where or what that element is. It also bundles in some semantic meaning with that classname.